fili-core/src/main/java/com/yahoo/bard/webservice/web/apirequest/ApiRequestImpl.java

```diff
@@ -627,11 +605,11 @@ protected static void validateTimeAlignment(
      * @return Response format type (CSV or JSON).
      * @throws BadApiRequestException if the requested format is not found.
      */
-    protected ResponseFormatType generateAcceptFormat(String format) throws BadApiRequestException {
+    protected DefaultResponseFormatType generateAcceptFormat(String format) throws BadApiRequestException {
         try {
-            return format == null ?
-                    ResponseFormatType.JSON :
-                    ResponseFormatType.valueOf(format.toUpperCase(Locale.ENGLISH));
+            return format == null || "".equals(format) ?
```

Comment:
Why are we adding case of `""`? I assume format is null is request string doesn't put format in it

Reply:
This is more of an abundance of precaution in case a container replaced a null with an "" for a parameter unspecified.  Probably unnecessary.